traverser/todoerrs/todoerrs.go

```diff
@@ -32,7 +33,10 @@ func commentsCallback(chk *checker.Checker, customTodos []string, todoErrCallbac
 		if err != nil {
 			return fmt.Errorf("couldn't check todo line: %w", err)
 		} else if todoErr != nil {
-			todoErrCallback(todoErr)
+			err = todoErrCallback(todoErr)
+			if err != nil {
+				logger.Info("couldn't run todo error callback: " + err.Error())
```

Comment:
We should instead return `fmt.Errorf("received error from todo err callback: %w", err)`

Reply:
I updated the code according to your suggestion.